webui/react/src/components/CreateGroupModal.tsx

```diff
@@ -24,38 +24,48 @@ import {
 import { V1GroupDetails, V1GroupSearchResult } from 'services/api-ts-sdk';
 import determinedStore from 'stores/determinedInfo';
 import roleStore from 'stores/roles';
-import { DetailedUser, UserRole } from 'types';
+import { UserRole } from 'types';
 import handleError, { ErrorType } from 'utils/error';
-import { getDisplayName } from 'utils/user';
 
 export const MODAL_HEADER_LABEL_CREATE = 'Create Group';
 export const MODAL_HEADER_LABEL_EDIT = 'Edit Group';
 const GROUP_NAME_NAME = 'name';
 export const GROUP_NAME_LABEL = 'Group Name';
 const GROUP_ROLE_NAME = 'roles';
-const GROUP_ROLE_LABEL = 'Global Roles';
-const USERS_NAME = 'users';
-export const USERS_LABEL = 'Users';
-const ADD_USERS = 'addUsers';
-const REMOVE_USERS = 'removeUsers';
+export const GROUP_ROLE_LABEL = 'Select Global Roles';
 export const API_SUCCESS_MESSAGE_CREATE = 'New group has been created.';
 const API_SUCCESS_MESSAGE_EDIT = 'Group has been updated.';
+const API_FAILURE_MESSAGE_CREATE = 'Error creating new group.';
+const API_FAILURE_MESSAGE_EDIT = 'Error editing group.';
 const FORM_ID = 'create-group-form';
 
+const CREATE_VALUES = {
```

Comment:
Thats nice! I think its good to have a type for `CREATE_VALUES` and `EDIT_VALUES` to prevent them to have different properties in the future change